packages/adapter-cloudflare-workers/README.md

```diff
@@ -14,16 +16,51 @@ Generate this file using `wrangler` from your project directory
 $ wrangler init --site my-site-name
 ```
 
-Then configure your sites build directory in the config file:
+Add the adapter to your `svelte.config.js`:
+
+```js
+import adapter from '@sveltejs/adapter-cloudflare-workers';
+const config = {
```

Comment:
If you make this `export default ...` then you don't need the `...` at the bottom
```suggestion
export default {
```